test/RewardPool-test.js

```diff
@@ -372,6 +406,11 @@ contract("RewardPool", function (accounts) {
           paymentCycle
         );
         root = merkleTree.getHexRoot();
+        rewardPool.submitPayeeMerkleRoot.estimateGas(root, { from: tally }).then(
+          function(gas) {
```

Comment:
All this lint stuff is [prettier](https://prettier.io/), ideally your editor should run on save but not sure what editor you're using or how much of a yak shave it is to set that up

Reply:
Perfect, wanted this but didn't want to run a different thing to everyone else. Is easy to setup on save, will do that.